packages/quantic/cypress/e2e/default-2/recent-results-list/recent-results-list.cypress.ts

```diff
@@ -140,4 +141,30 @@ describe('quantic-recent-results-list', () => {
       });
     });
   });
+
+  describe('when there are no results', () => {
```

Comment:
```suggestion
  describe('when #hideWhenEmpty is set to false', () => {
      it('should display an empty list when there are no results', () => {
          ...
```

What do you think of inverting those?

Reply:
Just an idea

Reply:
applied!